.github/workflows/build_deploy_docs_release.yml

```diff
@@ -59,10 +59,12 @@ jobs:
           git config user.name github-actions[bot]
           git config user.email 41898282+github-actions[bot]@users.noreply.github.com
 
-      - name: Rename folder and redirect stable to new release
+      - name: Rename folder, commit and redirect stable to new release
         run: |
             echo "Renaming site folder and committing"
+            git fetch
             git checkout gh-pages
+            rm -r -f "$RELEASE_VERSION"
```

Comment:
For the version we could also use: `${{ github.event.release.tag_name }}` which is nicer I think then getting it from the branch/tag name

Reply:
Done in new combined action for development and release